### ISSUE TITLE
error while running "packnet"

### ISSUE THREAD

**Issue by HyunghoNa** [Lifelong-Robot-Learning/LIBERO#97]
Hi everyone.

when defining **eval_algo** at packnet.py in the following code, I confront the NoneType error

<img width="573" height="361" alt="Image" src="https://github.com/user-attachments/assets/c21e4f57-9906-4d53-b539-d24cd24b30ee" />

 def get_eval_algo(self, task_id):
        # TODO: find a better way to do this
        # save and load a new model and set all params where mask > current_task + 1 to 0
        torch_save_model(
            self.policy,
            os.path.join(self.experiment_dir, "tmp_model.pth"),
            cfg=self.cfg,
        )
        eval_algo = safe_device(
            eval(self.cfg.lifelong.algo)(
                eval(self.cfg.benchmark_name)().n_tasks, self.cfg
            ),
            self.cfg.device,
        )
        model_state_dict, _, _ = torch_load_model(
            os.path.join(self.experiment_dir, "tmp_model.pth")
        )
        eval_algo.policy.load_state_dict(model_state_dict)

Does anyone resolve this issue?
